oauthenticator/tests/mocks.py

```diff
@@ -140,30 +140,17 @@ def access_token(request):
         Replies with JSON model for the token.
         """
         assert request.method == 'POST', request.method
-        if token_request_style == 'json':
-            body = request.body.decode('utf8')
-            try:
-                body = json.loads(body)
-            except ValueError:
-                return HTTPResponse(
-                    request=request,
-                    code=400,
-                    reason="Body not JSON: %r" % body,
-                )
-            else:
-                code = body['code']
```

Comment:
This section was removed and only influenced tests of auth0 that had been sent JSON in the Post body before, but now sends an urlencoded body like other does.